src/PhpSpec/ObjectBehavior.php

```diff
@@ -31,9 +31,90 @@
  * @method void beConstructedWith(...$arguments)
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments = array())
  * @method void beAnInstanceOf($class)
+ *
  * @method void shouldHaveType($type)
+ * @method void shouldNotHaveType($type)
+ * @method void shouldReturnAnInstanceOf($type)
```

Comment:
@stof agreed on that already. That was indeed a mistake of mine. But I would still like to wait for further opinions about having the other magic methods in `Subject` making expectations on SUT methods/properties. IMO it does not make sense having some in `ObjectBehaviour` but none in `Subject` (this seems weird). So as I stated before it should be all or nothing. The argument to say matchers are added at runtime and therefore should not be hard-coded / annotated also applies to `ObjectBehaviour` then...